### PR TITLE
IGAPP-670: Icons disappear when autorotate is on.

### PR DESCRIPTION
**Pull request by maxammann**
…s when rotating

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/App.tsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ import SnackbarContainer from './components/SnackbarContainer'
 import NetInfo from '@react-native-community/netinfo'
 import sendTrackingSignal from './services/sendTrackingSignal'
 import useSendOfflineJpalSignals from './hooks/useSendOfflineJpalSignals'
+import { enableScreens } from 'react-native-screens';
 
+enableScreens(true)
```
Comment (maxammann) [inline native/src/App.tsx:26]: This does nothing but it is good to state it explicitly imo

File: native/src/components/SimpleImage.tsx [hunk 2]
```diff
@@ -1,40 +1,32 @@
...
+import { Image, View, ImageSourcePropType } from 'react-native'
 import { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet'
 
 export type ImageSourceType = string | number | null
 type PropsType = {
   source: ImageSourceType
   style?: ViewStyleProp
 }
-const ThumbnailImage = styled(RNImage)`
-  flex: 1;
```
Comment (maxammann) [inline native/src/components/SimpleImage.tsx:13]: This was the issue

File: native/src/components/Tile.tsx [hunk 3]
```diff
@@ -35,11 +35,14 @@ class Tile extends React.Component<PropsType> {
 
   render(): ReactNode {
     const { tile, theme } = this.props
+    console.log(tile.thumbnail.toString())
```
Comment (klinzo) [inline native/src/components/Tile.tsx:38]:
```suggestion
```

File: native/src/components/Tile.tsx [hunk 4]
```diff
@@ -35,11 +35,14 @@ class Tile extends React.Component<PropsType> {
...
     return (
       <TileContainer>
         <TouchableOpacity onPress={this.onTilePress}>
-          <ThumbnailContainer source={tile.thumbnail} />
-          <TileTitle theme={theme}>{tile.title}</TileTitle>
+          <View>
```
Comment (klinzo) [inline native/src/components/Tile.tsx:42]: Is this change actually necessary? Seems to work the same way if not wrapped in a `View`.

**Review by klinzo — APPROVED** [review]
Tested :+1:

**Review by SteffiStuffel — APPROVED** [review]
